### PR TITLE
Fix #822 by updating the counts of the user

### LINKED ISSUE #822: 🐛 [Bug] load more is sometimes wrong on User Profile in the three Tabs
> [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-06-13T16:49:48Z" title="Thursday, June 13th 2019, 6:49:48 pm +02:00">Jun 13, 2019</time>_
_Closed <time datetime="2019-07-15T22:45:03Z" title="Tuesday, July 16th 2019, 12:45:03 am +02:00">Jul 16, 2019</time>_
---

## :bug: Bugreport

At the Users Profile sometimes the `load more` is shown, even when there is nothing to load!

### Steps to reproduce the behavior
1. login as Jenny Rostock
2. go to Users Profile
3. delete a Post of yourself
4. switch between Tabs
5. suddenly the `load more` showed up at the Posts and Comments Tab

Bildschirmfoto 2019-06-13 um 18 28 46

Bildschirmfoto 2019-06-13 um 18 28 36

### PR DESCRIPTION
**Pull request by Tirokk**
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-13T21:04:51Z" title="Saturday, July 13th 2019, 11:04:51 pm +02:00">Jul 13, 2019</time>_
_Merged <time datetime="2019-07-15T22:45:03Z" title="Tuesday, July 16th 2019, 12:45:03 am +02:00">Jul 16, 2019</time>_
---

The load more button depends on the total of the posts available.
Removing posts changes the total counts, so we have to manually update
it.

### Issues

- fixes #822 

### Todo

- [X] None

### CODE REVIEW

**Comment by Tirokk** [comment]
> [codecov[bot]](https://github.com/apps/codecov) **Authored by [codecov[bot]](https://github.com/apps/codecov)**
_<time datetime="2019-07-13T21:56:35Z" title="Saturday, July 13th 2019, 11:56:35 pm +02:00">Jul 13, 2019</time>_

---


# [Codecov](https://codecov.io/gh/Human-Connection/Human-Connection/pull/1041?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@4c91d8f`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

File: webapp/pages/profile/_id/_slug.vue [hunk 1]
```diff
@@ -331,6 +331,10 @@ export default {
     },
   },
   methods: {
+    removePostFromList(index) {
+      this.activePosts.splice(index, 1)
```
Comment (Tirokk) [inline webapp/pages/profile/_id/_slug.vue:335]: > [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-15T06:49:29Z" title="Monday, July 15th 2019, 8:49:29 am +02:00">Jul 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -331,6 +331,10 @@ export default {
     },
   },
   methods: {
+    removePostFromList(index) {
+      this.activePosts.splice(index, 1)
```


From where you get the `index` parameter, dear? @roschaefer 
After you deleted it from the `for loop` above …

Comment (Tirokk) [reply webapp/pages/profile/_id/_slug.vue:335]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-15T16:15:54Z" title="Monday, July 15th 2019, 6:15:54 pm +02:00">Jul 15, 2019</time>_

---


~The index parameter is still there! You pass the function `removePostFromList` and it has the signature of expecting one argument. When it's called it gets called with the required index param.~

@Tirokk you are absolutely right! I was assuming that `removePostFromList` was emitting the `index`, but no, that's the task of the page component. Stupid JS didn't crash but removed the first post in the list always.

**Review by Mogge — APPROVED** [review]
seems straightforward @roschaefer, and I've tested it and it seems to work as expected, unless I'm missing something

**Pull request closed without merging**